src/client/src/apps/MainRoute/widgets/spotTile/epics/rfqEpics.ts

```diff
@@ -78,20 +90,38 @@ export const rfqReceivedEpic: ApplicationEpic = action$ =>
   action$.pipe(
     ofType<Action, RfqReceivedActionType>(TILE_ACTION_TYPES.RFQ_RECEIVED),
     mergeMap(action => {
+      const { currencyPair } = action.payload
       const cancel$ = action$.pipe(
         ofType<Action, RfqReceivedTimerCancellableType>(
           TILE_ACTION_TYPES.RFQ_REJECT,
-          TILE_ACTION_TYPES.RFQ_EXPIRED,
           TILE_ACTION_TYPES.RFQ_RESET,
         ),
-        filter(
-          cancelAction =>
-            cancelAction.payload.currencyPair.symbol === action.payload.currencyPair.symbol,
-        ),
+        filter(cancelAction => cancelAction.payload.currencyPair.symbol === currencyPair.symbol),
       )
 
       return timer(action.payload.timeout + 1000).pipe(
-        map(() => rfqExpired({ currencyPair: action.payload.currencyPair })),
+        mergeMap(() =>
+          merge(
+            of(rfqExpired({ currencyPair })),
+            timer(IDLE_TIME_MS).pipe(
+              mergeMap(() =>
+                from([
+                  setNotional({
+                    currencyPair: currencyPair.symbol,
+                    notional: {
+                      value: getDefaultNotionalValue(),
```

Comment:
Related to [Santiago's comment on your other PR](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/1180#discussion_r332540177) should this be `getDefaultInitialNotionalValue` so that NZDUSD resets to 10m?

Reply:
We want the prices to stream again after a reset, so if we reset to 10m, then, that won't be the case...